### DISCUSSION TITLE
7zip Version Number

### DISCUSSION THREAD

**Discussion by hms54** [chocolatey-community/chocolatey-packages#2484] [Q&A] [answered]
The current 7zip package lists the version as 24.6.0.   However, the current version number listed on the official 7zip website is 24.06.    I'm just curious why there is a discrepancy.

**Comment by corbob** [comment, marked as answer]
This is due to the version number normalization employed by Chocolatey CLI 2.0 and newer: https://docs.chocolatey.org/en-us/choco/features/version-number-normalization

**Comment by hms54** [comment]
Thanks.   Your answer will help understand how to read version numbers in the future.

-=Howard
